server/datastore/mysql/microsoft_mdm.go

```diff
@@ -9,34 +9,8 @@ import (
 	"github.com/jmoiron/sqlx"
 )
 
-// MDMWindowsGetEnrolledDevice receives a Windows MDM device id and returns the device information.
-func (ds *Datastore) MDMWindowsGetEnrolledDevice(ctx context.Context, mdmDeviceHWID string) (*fleet.MDMWindowsEnrolledDevice, error) {
```

Comment:
I deleted a few DB methods that were unnecessary/only used in tests. It's best to create a test helper if necessary (TBD if that still is once the tests are adjusted too, for now I just started on the DB methods without touching the tests, so all tests will fail but more than that it doesn't build in its current partially done state)